sql/core/src/test/scala/org/apache/spark/sql/sources/CreateTableAsSelectSuite.scala

```diff
@@ -170,20 +170,17 @@ class CreateTableAsSelectSuite extends DataSourceTest with SharedSparkSession {
     }
   }
 
-  test("disallows CREATE EXTERNAL TABLE ... USING ... AS query") {
+  test("allow CREATE EXTERNAL TABLE ... USING ... AS query if location is specified") {
```

Comment:
and here too. If this test happen to be broken, it's likely there's an issue in the current change.